cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -395,6 +395,7 @@ if(builtin_afterimage)
       INSTALL_COMMAND  ${CMAKE_COMMAND} -E copy_if_different libAfterImage.lib <INSTALL_DIR>/lib/.
       LOG_DOWNLOAD 1 LOG_CONFIGURE 1 LOG_BUILD 1 LOG_INSTALL 1 BUILD_IN_SOURCE 1
       BUILD_BYPRODUCTS ${AFTERIMAGE_LIBRARIES})
+    set(AFTERIMAGE_INCLUDE_DIR ${CMAKE_BINARY_DIR}/AFTERIMAGE-prefix/src/AFTERIMAGE)
```

Comment:
Why is this needed? Could you please explain? Thanks.

Reply:
asimage is not installed on Windows, so the include path must be set to its build location (it is only used at build time anyway)